tests/sentry/api/endpoints/test_organization_metrics_usage.py

```diff
@@ -0,0 +1,151 @@
+from sentry.testutils.cases import APITestCase
+
+
+class TestOrganizationMetricsUsage(APITestCase):
+    endpoint = "sentry-api-0-organization-metrics-usage"
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.login_as(user=self.user)
+
+        self.mri1 = "c:foo/1"
+        self.mri2 = "c:bar/2"
```

Comment:
nit: these are actually not valid MRIs. 😅
```suggestion
        self.mri1 = "c:custom/foo1@none"
        self.mri2 = "c:custom/bar2@none"
```